CONTRIBUTING.md

```diff
@@ -10,9 +10,10 @@ First, in terms of structure, there is no particular concept of "Bitcoin Core
 developers" in the sense of privileged people. Open source often naturally
 revolves around a meritocracy where contributors earn trust from the developer
 community over time. Nevertheless, some hierarchy is necessary for practical
-purposes. As such, there are repository maintainers who are responsible for
-merging pull requests, the [release cycle](/doc/release-process.md), and
-moderation.
+purposes. As such, there are repository [maintainers](/MAINTAINERS.md) who are
```

Comment:
why link to `MAINTAINERS.md` twice here?

Reply:
i guess it is redundant? can remove, i just default to linking to where terms are defined, and separately to where more can be read.